bigtable-client-core-parent/bigtable-client-core/src/test/java/com/google/cloud/bigtable/grpc/IntegrationBigtableSessionTest.java

```diff
@@ -45,7 +46,11 @@ public void connectivityTest() {
             .setUserAgent("Test")
             .build();
     try (BigtableSession bs = new BigtableSession(options)) {
-      bs.getTableAdminClientWrapper().listTables();
+      bs.getTableAdminClient()
+          .listTables(
+              ListTablesRequest.newBuilder()
+                  .setParent("projects/" + projectId + "/instances/" + instanceId)
```

Comment:
I would use the `BIGTABLE_V2_INSTANCE_FMT` utility in BigtableInstanceName

Reply:
done